src/components/map/Map.tsx

```diff
@@ -357,14 +489,25 @@ const styles = StyleSheet.create({
   myLocationButton: {
     alignItems: 'center',
     backgroundColor: colors.surface,
-    borderRadius: 50,
-    top: normalize(15),
+    borderRadius: normalize(50),
     height: normalize(48),
     justifyContent: 'center',
     position: 'absolute',
     right: normalize(15),
+    top: normalize(15),
     width: normalize(48),
     zIndex: 1
+  },
+  markerContainer: {
+    alignItems: 'center',
+    justifyContent: 'center'
+  },
+  activeMarkerContainer: {
+    // Define your styles for active marker here
+  },
```

Comment:
can be removed i guess?

Reply:
would also say so, will be removed in the upcoming commit.